config/main.py

```diff
@@ -1824,6 +1824,43 @@ def override_config_by(golden_config_path):
     return
 
 
+def generate_sysinfo(cur_config, config_input, ns=None):
```

Comment:
>generate_sysinfo [](http://example.com/codeflow?start=4&length=16)

Could you add some function level comments?
At least mention that this function will change `config_input`, which is important to code reader.